src/layout/section/heroSection/HeroSection.story.mdx

```diff
@@ -0,0 +1,130 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks'
+import { select, text } from '@storybook/addon-knobs'
+import { MediaSizeProvider } from '../../../_utils/mediaSizeProvider'
+import { AutoCompleteExample } from '../../../autoComplete/story'
+import { SearchForm } from '../../../searchForm'
+import { HeroSection } from './HeroSection'
+import carpoolBlaBlaBusLarge from './images-for-stories/carpool-bbb-large.svg'
+import carpoolBlaBlaBusSmall from './images-for-stories/carpool-bbb-small.svg'
+import carpoolBlueBusLarge from './images-for-stories/carpool-blue-bus-large.svg'
+import carpoolBlueBusSmall from './images-for-stories/carpool-blue-bus-small.svg'
+import carpoolCoralBusLarge from './images-for-stories/carpool-coral-bus-large.svg'
+import carpoolCoralBusSmall from './images-for-stories/carpool-coral-bus-small.svg'
+import carpoolOnlyLarge from './images-for-stories/carpool-large.svg'
+import carpoolOnlySmall from './images-for-stories/carpool-small.svg'
+
+<Meta title="Sections|HeroSection" />
+
+# **HeroSection**
+
+The hero section for a landing page.
+
+export const bottomElement = (
+  <SearchForm
+    onSubmit={() => {}}
+    autocompleteFromPlaceholder="Leaving From"
+    autocompleteToPlaceholder="Going to"
+    renderAutocompleteFrom={props => <AutoCompleteExample {...props} />}
+    renderAutocompleteTo={props => <AutoCompleteExample {...props} />}
+    datepickerProps={{
+      defaultValue: new Date().toISOString(),
+      format: value => new Date(value).toLocaleDateString(),
+    }}
+    stepperProps={{
+      defaultValue: 1,
+      min: 1,
+      max: 8,
+      title: 'Choose your number of seats',
+      increaseLabel: 'Increase the number of seats by 1',
+      decreaseLabel: 'Decrease the number of seats by 1',
+      format: value => `${value} seat(s)`,
+      confirmLabel: 'Submit',
+    }}
+  />
+)
+
+<Preview>
+  <Story name="Default">
```

Comment:
nit: maybe name it "Classic" instead of "Default" to reflect the component naming?

Reply:
I think SPA must define a convention for naming "base" stories. Nowadays, the ui-library has `Default, Basic, and Classic` as "base" namings

Reply:
So, which one do I use?